libgalois/include/katana/PropertyGraph.h

```diff
@@ -232,12 +232,30 @@ class KATANA_EXPORT PropertyGraph {
   }
 
   // TODO(witchel): ChunkedArray is inherited from arrow::Table interface but this is
-  // really a ChunkedArray of one change, change to arrow::Array.
+  // really a ChunkedArray of one chunk, change to arrow::Array.
+  const std::shared_ptr<arrow::ChunkedArray>& host_to_owned_global_ids() const {
+    return rdg_.host_to_owned_global_ids();
+  }
+  void set_host_to_owned_global_ids(std::shared_ptr<arrow::ChunkedArray>&& a) {
+    rdg_.set_host_to_owned_global_ids(std::move(a));
+  }
+
```

Comment:
Do we want to continue with `arrow::ChunkedArray`?

Reply:
Yes, I lean towards supporting them.